.gitignore

```diff
@@ -30,3 +30,5 @@ node_modules
 
 # Users Environment Variables
 .lock-wscript
+
+.screeps-creds
```

Comment:
Hmm I still need a good solution for this. Even though I have to fix it, I don't like to see these things in my commits...